tracer/src/Datadog.Trace/Ci/CiEnvironment/ManualParserGitInfoProvider.cs

```diff
@@ -0,0 +1,580 @@
+// <copyright file="ManualParserGitInfoProvider.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.Ci.CiEnvironment;
+
+/// <summary>
+/// Manual parser Git information class
+/// </summary>
+internal sealed class ManualParserGitInfoProvider : GitInfoProvider
```

Comment:
This is is basically what you already had right? I haven't bothered reviewing it again, but can if I'm wrong 🙂

Reply:
yup.